app/models/company.rb

```diff
@@ -0,0 +1,2 @@
+class Company < ApplicationRecord
```

Comment:
add the relation `branch-employee` in the models. you'll need migrations and specs too.

Reply:
done